contracts/modules/token/TokenController.sol

```diff
@@ -151,11 +151,21 @@ contract TokenController is ITokenController, LockHandler, MasterAwareV2 {
   }
 
   /**
-  * @dev Mints new token for an address
-  * @param _member address to reward the minted tokens
+  * @dev Mints new tokens for an address and checks if the address is a member
+  * @param _member address to send the minted tokens to
   * @param _amount number of tokens to mint
   */
   function mint(address _member, uint _amount) public override onlyInternal {
+    _mint(_member, _amount);
+  }
+
+  /**
+  * @dev Internal function to mint new tokens for an address and checks if the address is a member
```

Comment:
is it worth adding a comment here that internal functions should only use `_mint` and never `token.mint` directly?

so that future devs are aware and don't make mistakes